### PR TITLE
Adding raspberry pi linux service

### PR DESCRIPTION
**Pull request by itay-sho**
important in case of RPI shutdown etc

### CODE REVIEW

File: 1.raspberry-pi-stuff/service/octopus.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Octopus Recurse application website
+After=network.target
+
+[Service]
+User=root
+Enviroment="FLASK_SECRET_KEY=very very secrete"
```
Comment (itay-sho) [inline 1.raspberry-pi-stuff/service/octopus.service:7]: this is probably not best practice. we might want to add better logic in install.sh to write it down on the installation and save it in a conf file

File: 1.raspberry-pi-stuff/service/octopus.service [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+WorkingDirectory=/root
+ExecStart=hypercorn octopus:app --bind 0.0.0.0:5555
```
Comment (itay-sho) [inline 1.raspberry-pi-stuff/service/octopus.service:9]: I am not sure what we were speaking about where we want to host the website itself.
but regardless this line is probably missing the internal raspberry pi related services

File: 1.raspberry-pi-stuff/service/octopus.service (hunk 1 above)
Comment (gregsadetsky) [reply 1.raspberry-pi-stuff/service/octopus.service:7]: agreed but yes, we'll switch to soooooome sort of secret environment thingamabob at some point??

(((((((((((((could we deploy to the raspi from github actions???????)))))))))))))))))))))

**Review by gregsadetsky — APPROVED** [review]

**Comment by gregsadetsky** [comment]
Sorry @itay-sho it doesn't look like I merged this? Maybe because it's a draft PR..? It's merge ready or you'd want to keep contributing to it before it's merged?

Thanks!

**Comment by gregsadetsky** [comment]
right -- it's a draft PR (see below).

lmk if you want it merged or not! thx

image

**Comment by itay-sho** [comment]
Yeah :) let's merge it
thanks!

**Pull request merged**